crates/distribution-types/src/prioritized_distribution.rs

```diff
@@ -24,6 +24,8 @@ struct PrioritizedDistInner {
 /// A distribution that can be used for both resolution and installation.
 #[derive(Debug, Clone)]
 pub enum CompatibleDist<'a> {
+    /// The distribution is already installed and can be used.
+    InstalledDist(Dist),
```

Comment:
Clippy is upset about the size of this variant and it would make sense for it to be borrowed like the rest, but I'm not sure how to create a borrowed distribution in `CandidateSelector`.